src/backend/marsha/core/api/pairing_challenge.py

```diff
@@ -67,5 +67,8 @@ def pairing_challenge(request):
     Device.objects.get_or_create(id=request.data.get("box_id"))
 
     return Response(
-        {"jitsi_url": f"https://{settings.JITSI_DOMAIN}/{live_pairing.video.id}"}
+        {
+            "jitsi_url": f"https://{settings.JITSI_DOMAIN}/{live_pairing.video.id}",
+            "endpoints": live_pairing.video.get_medialive_input().get("endpoints"),
```

Comment:
Maybe call it something coherent with `jitsi_url`? `rtmp_urls` or `rtmp_endpoints`?
```suggestion
            "rtmp_urls": live_pairing.video.get_medialive_input().get("endpoints"),
```

Otherwise it looks GTM but @lunika must first confirm that this info is sufficient to authenticate on the RTMP endpoint?